### PR TITLE
First draft of main installation re-write

### PR DESCRIPTION
**Pull request by noccor**
This is a first draft of a re-write for the OTel main installation page. 

Changes made:

1. 'an app' becomes 'your app' ~> why? Because of the ambiguity between app's in AppSignal and web apps. 
2. Directional text for code snippets ~> Adding "In the example below" instructs users using assistive technologies (like text to speech) on where things are in the code, so they can also skim read etc. 
3. Slight tone softening 
4. Tried to extract requirements hidden in text, to make them more explicit. This likely will need some work, as I'm not 100% sure of it.

### CODE REVIEW

**Review by unflxw — APPROVED** [review]
Changes looks good so far! Left some nitpicks. Some things you may want to know:

- When you push changes to this repo, Netlify or Vercel or whatever [builds a deploy preview](https://deploy-preview-628--dreamy-wright-e9a95c.netlify.app/nodejs/installation/opentelemetry.html), which allows you to take a proper look at what your changes will look like on the site. It's the bee's knees! 🐝 
- Our docs' Markdown has support for "notice" and "warning" styled boxes (there may also be other box styles I'm not aware of) which you may want to use, both to make them more visible, and to break the visual monotony of the docs -- [here's what they look like in the docs](https://docs.appsignal.com/nodejs/installation/) (search for "auto-instrument"), and [here's what they look like in the source](https://github.com/appsignal/appsignal-docs/blob/main/source/nodejs/installation/index.html.md?plain=1#L85-L93).

Something that also needs to be fixed in the OpenTelemetry docs (in this PR or a separate one) is [the issue we discussed on Slack](https://appsignal.slack.com/archives/CNPP953E2/p1657718094357949?thread_ts=1657715802.879699&cid=CNPP953E2), where our docs tell you to write `instrumentations: [WhateverInstrumentation]` instead of `instrumentations: [new WhateverInstrumentation()]`.

On the Basecamp post, you mentioned you wanted to extract implicit requirements hidden in the text and make them visible. This PR does a great job so far, feel free to ping us if we can help with that further!

File: source/nodejs/installation/opentelemetry.html.md [hunk 1]
```diff
@@ -25,19 +28,23 @@ Our integration supports the following libraries from OpenTelemetry instrumentat
...
 Additional instructions for packages, if necessary, are listed in the [instrumenting packages](#instrumenting-packages) section.
 
-Support for more packages will follow later.
+## Application Requirements
+
+Currently, an app will need a working AppSignal integration for one of the following frameworks to create an AppSignal root span:
+
+* Express
+* Koa.js
+* or Apollo Server
```
[the remark this answers is not in the document]
Comment (noccor) [reply source/nodejs/installation/opentelemetry.html.md:37]: Noted, we can drop that then

File: source/nodejs/installation/opentelemetry.html.md [hunk 2]
```diff
@@ -46,14 +53,16 @@ const { Appsignal } = require("@appsignal/nodejs");
...
 });
 
 // Export the AppSignal client to use in other files
 module.exports = { appsignal };
 ```
 
-Require `appsignal.js` in the app at the top of the main app file. Make sure another integration for a web framework, like [Express](/nodejs/integrations/express.html), is configured.
+Require `appsignal.js` in your app at the top of the main app file. Make sure another integration for a web framework, like [Express](/nodejs/integrations/express.html), is configured.
+
+Ensure `appsignal.js` is required in your `index.js` file (the name may vary per project - this is the file you call to run your server). Another web framework integration should also be configured, like Express, like in the example below of an `index.js` file.
```
[the remark this answers is not in the document]
Comment (noccor) [reply source/nodejs/installation/opentelemetry.html.md]: I have removed, and re-worded the line to be more explicit about what they need regarding web framework integration - that it must be A required one, linking them back up to the requirements section I'm sure they didn't skip past.

**Review by jeffkreeftmeijer — APPROVED** [review]
Looking great! Agreed with Noemi’s comments. :+1:

**Review by roytomeij — APPROVED** [review]
Approved, but still good to look at my comments ;)

File: source/nodejs/installation/opentelemetry.html.md [hunk 3]
```diff
@@ -126,17 +139,17 @@ In the app's `package.json`, update the way the app is started to require the `t
 }
 ```
 
-Now AppSignal and OpenTelemetry are integrated in the app and with each other. Add instrumentation for any packages in the next section. When you're done with the configuration, start the app to start receiving events from OpenTelemetry from the [libraries we support](#supported-libraries).
+🎉  You've now configured your app to use AppSignal with OpenTelemetry! Next, you can add instrumentation for the [libraries we support](#supported-libraries). When you've completed your configuration, start your app to receive events from OpenTelemetry in AppSignal.
```
[the remark this answers is not in the document]
Comment (noccor) [reply source/nodejs/installation/opentelemetry.html.md]: Good point, altered text

File: source/nodejs/installation/opentelemetry.html.md [hunk 4]
```diff
@@ -111,12 +122,14 @@ tracer.addSpanProcessor(new SpanProcessor(appsignal));
 tracer.register();
 ```
 
-#### Updating `package.json` scripts
+#### Updating `package.json` Scripts
+
+The `tracing.js` file is required whenever your app starts. To ensure tracing is automatically loaded, open the `package.json` file and in the scripts section, add `--require ./tracing.js` property to a script that runs your node.js server (see example below). Doing this ensures tracing is always loaded first in your app and that a manual require of `tracing.js` is not always needed.
```
[the remark this answers is not in the document]
Comment (noccor) [reply source/nodejs/installation/opentelemetry.html.md]: I don't think this text is needed tbh, it makes an assumption the user will manually include the file on start of their node app... but our installation guide is set up so they don't have to do that and does not cover any scenario when a manual require is needed. So I've removed this reference - good find Roy!

File: source/nodejs/installation/opentelemetry.html.md [hunk 5]
```diff
@@ -2,19 +2,22 @@
...
 
-The AppSignal Node.js integration has experimental support for [OpenTelemetry](https://opentelemetry.io/) trace data. Our `@appsignal/nodejs` package ships with an OpenTelemetry processor that can transform OpenTelemetry trace data into AppSignal trace data to send to AppSignal.com. This is our first step in supporting the OpenTelemetry ecosystem in AppSignal.
+-> The first release with OpenTelemetry support is `@appsignal/nodejs` package 2.4.0.
 
-Please [let us know][contact] if you have trouble setting up the MySQL integration, if you're seeing unexpected results, or if you have any feedback on this integration. We're here to help!
+The AppSignal Node.js integration supports OpenTelemetry trace data. Our `@appsignal/nodejs` package ships with an OpenTelemetry processor that converts OpenTelemetry trace data to AppSignal trace data and sends it to AppSignal.
 
--> The first release with OpenTelemetry support is `@appsignal/nodejs` package 2.4.0.
+
+**🔧 Setting up the MySQL integration?** Please let us know if you're seeing unexpected results or have any feedback on this integration. [We're here to help!](https://www.appsignal.com/contact)
```
[the remark this answers is not in the document]
Comment (noccor) [reply source/nodejs/installation/opentelemetry.html.md:10]: @luismiramirez or @shairyar ~ could you drop some context here?

**Pull request merged**